.project-keeper.yml

```diff
@@ -1,4 +1,4 @@
 sources:
   - type: golang
     path: go.mod
-version: 0.4.6
+version: 0.4.7
```

Comment:
Can you change also the version in `version.go ` that the test is green again

Reply:
✅